app/lib/reports/careplus_exporter.rb

```diff
@@ -140,11 +140,31 @@ def vaccine_fields(vaccination_records, index)
 
     [
       record.vaccine.snomed_product_code, # Vaccine X
-      "", # Dose X field; Not sure, documentation says this is derived later?
+      "1P", # Dose X field; Apparently 1P or 2P for HPV, but not sure why/when it changes
       VaccinationRecord.human_enum_name(:reason, record.reason), # Reason Not Given X
-      record.delivery_site, # Site X; Coded value, but we don't know the codes yet
+      coded_site(record.delivery_site), # Site X; Coded value
       record.vaccine.manufacturer, # Manufacturer X
       record.batch.name # Batch No X
     ]
   end
+
+  def coded_site(site)
+    {
+      # These are confirmed
+      left_arm: "LA",
+      right_arm: "RA",
+      # These are made up/guessed
+      left_arm_upper_position: "LUP",
```

Comment:
I think we really should check this first

Reply:
Should we merge and iterate later?

Reply:
@benilovj I've temporarily disabled the Careplus format via a setting.